OpenRA.Mods.Common/Traits/Multipliers/HandicapDamageMultiplier.cs

```diff
@@ -0,0 +1,40 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Modifies the damage applied to this actor based on the owner's handicap.")]
+	public class HandicapDamageMultiplierInfo : TraitInfo
+	{
+		public override object Create(ActorInitializer init) { return new HandicapDamageMultiplier(init.Self); }
+	}
+
+	public class HandicapDamageMultiplier : IDamageModifier
+	{
+		readonly Actor self;
+
+		public HandicapDamageMultiplier(Actor self)
+		{
+			this.self = self;
+		}
+
+		int IDamageModifier.GetDamageModifier(Actor attacker, Damage damage)
+		{
+			// Equivalent to the health handicap from C&C3:
```

Comment:
Doesn't this modifier's effectiveness scale with the number of attackers? And is it really equivalent to a reduced healthpool on the receiving end?

Reply:
`IDamageModifier` is for damage received, so yes it is equivalent to a reduced health pool.

Reply:
My thought was that nerfing a unit's health by 10% is somehow different than buffing all the enemies damage output against it by 10%...because the 10% more damage can multiply with the number of things shooting at the unit...and when nerfing health by 10%, it does not matter if one or many units drain health.

Reply:
This isn't buffing enemy damage output - that is controlled by `IFirepowerModifier`.

Reply:
I'm probably wrong about it anyways. (I can't close code comments.)